ml-dsa/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "ml-dsa"
```

That's squatted by Tony already. Let's always prefix with `libcrux`.

```suggestion
name = "libcrux-ml-dsa"
```